package/lib/compileFunctions.js

```diff
@@ -57,6 +59,15 @@ module.exports = {
         });
       }
 
+      if (vpcEgress) {
```

Comment:
It looks like now it ignores the setting on `function` level, is that the case or am I missing something here? It should override provider-level setting with function-specifc setting